client/state/posts/selectors.js

```diff
@@ -359,6 +363,31 @@ export const isEditedPostDirty = createSelector(
 );
 
 /**
+  * Returns true if the edited post has content
+  * (title, excerpt or content not empty)
+  *
+  * @param  {Object}  state  Global state tree
+  * @param  {Number}  siteId Site ID
+  * @param  {Number}  postId Post ID
+  * @return {Boolean}        Whether the edited post has content or not
+ */
+export function editedPostHasContent( state, siteId, postId ) {
+	const editedPost = getEditedPost( state, siteId, postId );
+	return (
+		!! editedPost &&
+		(
+			some( [ 'title', 'excerpt' ], ( field ) => editedPost[ field ] && !! editedPost[ field ].trim() ) ||
+			/*
```

Comment:
Nitpick: Per guidelines, multi-line comments which are not docblocks should use `//` ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#comments))